glib/sys/src/lib.rs

```diff
@@ -2195,20 +2195,6 @@ pub struct _GStatBuf {
 
 pub type GStatBuf = _GStatBuf;
 
-#[derive(Copy, Clone)]
-#[repr(C)]
-pub struct GStaticMutex {
-    pub mutex: *mut GMutex,
-}
-
-impl ::std::fmt::Debug for GStaticMutex {
-    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
-        f.debug_struct(&format!("GStaticMutex @ {self:p}"))
-            .field("mutex", &self.mutex)
-            .finish()
-    }
-}
-
 #[derive(Copy, Clone)]
 #[repr(C)]
 pub struct GStaticPrivate {
```

Comment:
This struct should be ignored as well, sorry for not noticing earlier. Otherwise lgtm

Reply:
Comment addressed.